test/junit/src/test/scala/strawman/collection/mutable/VectorTest.scala

```diff
@@ -29,20 +30,13 @@ class VectorTest {
   )
 
   @Test
-  def vectorCat() {
+  def vectorCat(): Unit = {
     val cats = vecs.map(a => vecs.map(a ++ _))
     assert( cats == ans )
   }
 
   @Test
-  def iteratorCat() {
-    def its = vecs.map(_.toList.toIterator)
-    val cats = vecs.map(a => its.map(a ++ _))
-    assert( cats == ans )
-  }
```

Comment:
I removed this one because `concat` now takes an `Iterable`, not an `IterableOnce`.